templates/zerver/plans.html

```diff
@@ -87,14 +87,8 @@ <h2>Premium</h2>
                                     <button class="black-disabled" type="button">
                                         Current plan
                                     </button>
-                                    {% elif realm_plan_type == 2 %}
-                                    <a href="/upgrade">
-                                        <button class="green" type="button">
-                                            Buy Premium
-                                        </button>
-                                    </a>
                                     {% else %}
-                                    <a href="/new">
+                                    <a href="/upgrade">
```

Comment:
Can you rebase to resolve this merge conflict (I think just s/Premium/Standard/)